docker/jenkins.yml

```diff
@@ -52,7 +52,7 @@ jenkins:
   - ecs:
       allowedOverrides: "all"
       cluster: "${/mgmt/jenkins_cluster_arn}"
-      jenkinsUrl: "${/mgmt/jenkins_load_balancer_url}"
+      jenkinsUrl: "http://10.0.0.167"
```

Comment:
If we can't pick this up automatically, it'd be good to add a comment pointing to the Terraform config where this is configured (and vice-versa).

Reply:
I've added this in as an ssm parameter and references the parameter instead. It's cleaner than having the IP in there.